cl/alerts/api_serializers.py

```diff
@@ -28,23 +33,60 @@ class Meta:
     def validate(self, attrs):
         """Validate the query type and set default for alert_type if not
         provided."""
-        if "query" not in attrs:
+
+        query = attrs.get("query", "")
+        if not query:
             return attrs
```

Comment:
I'm wondering if this scenario is actually possible. My understanding is that serializers first run field-specific validators, and then execute the `validate` method. The `query` field is defined as **required** and cannot be **blank**. I confirmed this by sending a request to the `/alerts` endpoint without the `query` field, which returned `'This field is required'` Unless there's a way to use this serializer without providing a value for the `query` field, I suggest we remove the `if` statement, as it implies the query field could be `null` or not included in the body of the request.